packages/traceability/issuer-api/src/pods/certificate/certificate.entity.ts

```diff
@@ -61,4 +60,10 @@ export class Certificate extends ExtendedBaseEntity {
     @Column()
     @IsBoolean()
     issuedPrivately: boolean;
+
+    @CreateDateColumn({ type: 'timestamptz' })
+    createdAt: Date;
+
+    @UpdateDateColumn({ type: 'timestamptz' })
+    updatedAt: Date;
```

Comment:
Any reason for this change?

Reply:
ExtendedBaseEntity which implements createdAt and updatedAt is ActiveRecord in fact, and either I create `ICertificate` or something like that, or make this entity not ActiveRecord. We don't make use of this entity being the ActiveRecord, we use repository pattern everywhere.

The problem with ActiveRecord is that it cannot be easily created as an object (`{ ... }`)

Reply:
Cool, I remember now. I'm fine with this.